data/tooltips.txt

```diff
@@ -497,6 +497,12 @@ tip "shield generation:"
 tip "shield generation multiplier:"
 	`Modifies the amount of shield points recharged per second by the given factor.`
 
+tip "full shield permeability:"
+	`This percentage of all incoming damage bypasses shields and applies directly to the hull when at full shields.`
+
+tip "low shield permeability:"
+	`This percentage of all incoming damage will bypass shields and apply directly to the hull as shields approach 0 (in practice, this number is never reached, because when shields are 0 all damage goes to the hull).`
```

Comment:
We don't need to tell player stuff like this. Active cooling, for example, doesn't give this same caveat. 
```suggestion
	`This percentage of all incoming damage will bypass shields and apply directly to the hull as shields approach zero.`
```